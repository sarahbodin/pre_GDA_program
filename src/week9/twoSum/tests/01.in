6
14 0,1,5,7,9
3 1,1,1
4 80,2
-4 -3,5,9,-1
2 10,-1,8,-8
0 -5,8,0,-2
