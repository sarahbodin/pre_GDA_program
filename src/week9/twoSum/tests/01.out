Case #1: true
Case #2: false
Case #3: false
Case #4: true
Case #5: true
Case #6: false
